compiler/src/dotty/tools/dotc/Compiler.scala

```diff
@@ -109,6 +109,9 @@ class Compiler {
            new LabelDefs),          // Converts calls to labels to jumps
       List(new GenBCode)            // Generate JVM bytecode
     )
+    assert(allPhases.last.last.id <= Periods.MaxPossiblePhaseId)
```

Comment:
The idea is that compiler plugins may (in future) override\filter\change phases somehow. This means that this is wrong place to test the phase-plan, because it may still change. Phases have not been combine or initialized.

Additionally, Transforms require their own periods, and they will not be set here.